examples/svelte-apollo/src/AddArticles.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+  import { mutation } from 'svelte-apollo'
+  import { gql } from '@apollo/client'
+
+  const ARTICLE_LIST = gql`
+    query {
+      article(order_by: [{ title: asc }]) {
+        title
+      }
+    }
+  `
+  const ADD_ARTICLE = gql`
+    mutation ($title: String!, $content: String!, $author_id: Int!) {
+      insert_article(objects: [{ title: $title, content: $content, author_id: $author_id }]) {
+        affected_rows
+      }
+    }
+  `
+  let title = ''
+  let content = ''
+  let author_id = ''
+
+  const mutate = mutation(ADD_ARTICLE)
+  async function addArticles(e) {
```

Comment:
addArticles -> addArticle as only a single article is being added by the form.

Reply:
Done